lms/templates/credit_notifications/credit_eligibility_email.html

```diff
@@ -34,7 +34,17 @@
             </p>
 
             <p>
-              ${_(u"Congratulations! You are eligible to receive course credit for successfully completing your {platform_name} course! {link_start}Get your credit now.{link_end}").format(
+               ${_(u"Congratulations! You are eligible to receive course credit from ")}
+                % if providers:
+                     <ul>
+                       % for provider in providers:
+                        <li>
+                            ${ provider }
+                        </li>
+                       % endfor
+                     </ul>
+                % endif
+               ${_(u" for successfully completing your {platform_name} course! {link_start}Get your credit now.{link_end}").format(
```

Comment:
I think `{link_start}Get your credit now.{link_end}` would be different for each provider. So I guess we would need to have a different text here.